pkg/volume/testing/OWNERS

```
@@ -1,14 +1,10 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
+- sig-storage-approvers
```

Should we let this bubble up to pkg/volume/OWNERS?

---

Sure.